src/sentry/replays/usecases/delete.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
```

Comment:
i know we have code existing for replays deletions, is any of it able to be reused?